src/sortStudents.ts

```diff
@@ -1,16 +1,58 @@
-
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
+
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  const result = [...students];
+
+  const averageGrade = (array: number[]): number => array
+    .reduce((prev, current) => prev + current, 0) / array.length;
+
+  result.sort((first, second) => {
```

Comment:
тут можна скоротити ініціалізацію, присвоєння змінної, сорутвання та повернення)
```suggestion
  return [...students].sort((first, second) => {
```